android/KMAPro/build.sh

```diff
@@ -109,3 +113,15 @@ if builder_start_action test; then
 
   builder_finish_action success test
 fi
+
+if builder_start_action publish; then
+  # Copy Release Notes
+  generateReleaseNotes
+
+  # Publish Keyman for Android to Play Store
+  echo "PUBLISH_FLAGS $PUBLISH_FLAGS"
+  cd "$KEYMAN_ROOT/android/KMAPro/"
+  ./gradlew $DAEMON_FLAG $PUBLISH_FLAGS
```

Comment:
I wonder if `$PUBLISH_FLAGS` is necessary if it never changes? Just put it inline in the script and reduce the set of variables floating about.

```suggestion
  ./gradlew $DAEMON_FLAG publishReleaseApk
```